custom_model_runner/datarobot_drum/drum/common.py

```diff
@@ -361,3 +363,10 @@ def verify_pyarrow_module():
     if pyarrow is None:
         raise ModuleNotFoundError("Please install pyarrow to support Arrow format")
     return pyarrow
+
+
+def generate_temp_target_name(target_name, colnames):
```

Comment:
this might be overkill?

Reply:
Yeah maybe we should just do something like __target__ here.